Resources/Private/JavaScript/asset-upload/src/components/FilePreview.module.css

```diff
@@ -1,25 +1,20 @@
-.fileList {
-    margin-top: var(--theme-spacing-Full);
+.preview {
     display: flex;
     flex-direction: row;
-    flex-wrap: wrap;
-}
-
-.fileListHeader {
-    flex: 1 1 100%;
-    margin-bottom: var(--theme-spacing-Full);
-    font-size: var(--theme-fontSize-base);
+    column-gap: 1rem;
+    padding-bottom: 1rem;
+    border-bottom: 1px solid #3f3f3f;
 }
 
 .thumb {
     display: inline-flex;
     border-radius: 2px;
     border: 1px solid #eaeaea;
-    margin-bottom: var(--theme-spacing-Half);
-    margin-right: var(--theme-spacing-Half);
-    width: 100px;
-    height: 100px;
-    padding: var(--theme-spacing-Quarter);
+    margin-bottom: var(--theme-spacing-Half, 8px);
```

Comment:
Why did you need to add those fallbacks everywhere?
The `Theme.module.css` should provide all necessary variables. If not then that's a bug that needs to be fixed wherever needed.

Reply:
`Theme.module.css` is missing in the Content view of the backend, therefore the fallback values are needed when the new upload screen is used in the content view.

Reply:
Then you are missing `className={theme.mediaModuleTheme}` there like in the Dialog.tsx and App.tsx.
But this dialog can anyway not be triggered right now without your Neos.UI change right?